zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/RestErrorMapperTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.camunda.service.exception.ServiceException;
+import io.camunda.service.exception.ServiceException.Status;
+import java.util.Optional;
+import java.util.concurrent.CompletionException;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ProblemDetail;
+import org.springframework.http.ResponseEntity;
+
+public class RestErrorMapperTest {
```

Comment:
❤️  great that you added a holistic test for the mapper!

Reply:
I do like the simplicity of this one for sure. Maybe there is a way to merge the efforts somehow.

Reply:
Also fine to keep the concerns separated, I guess. But it might also lead to confusion in the future regarding where to add tests.